### ISSUE TITLE
Command blocks: setting for command's number of lines

### ISSUE THREAD

**Issue by CharlesNepote** [wavetermdev/waveterm#149]
IMHO, one of the awesome and so innovative features of Wave terminal is these command blocks.

The issue I'm facing is the number of lines displayed in past command blocks. Often I need to see more than the 4 lines currently displayed -- I know I can scroll, but this is not very handy. Picture below.

![image](https://github.com/wavetermdev/waveterm/assets/562352/7f20aa87-b786-4cf9-b861-9a9e012cfc5b)

A setting could be great to let people adjust the number of lines, eg.: [1-99], [0 for infinity]. Personally, based on my usages, I guess I would choose 6 or 7.

**Comment by sawka** [comment]
Ah, this is something I never thought about making customizable, but I like it!  What type of granularity do you think would be most useful?  A global setting, a setting per tab/workspace, or a per block setting?

**Comment by CharlesNepote** [comment]
You're right I did not think about the granularity. I guess a per block setting would be too much. I'd say a global setting would be a good start.
I'm not sure a setting per tab would make a big difference.
